### DISCUSSION TITLE
Hide minimize button

### DISCUSSION THREAD

**Discussion by barnabaspal** [AvaloniaUI/Avalonia#7818] [Q&A]
Is it possible to hide or disable the minimize button of the window?
I found only the CanResize proeprty, which disables the maximize buttob, but I need to deactivate the minize button too:
![image](https://user-images.githubusercontent.com/97017869/158580938-ebb20bf3-f644-46de-ae37-40db4333c3eb.png)

**Comment by AntonNah** [comment]
it's too hard for avalonia

**Comment by sn4k3** [comment]
You can make your own "title bar" and render the buttons as you want. Many apps does that to fully integrate into style

**Reply by AlanPBourke** [reply]
I think this is the only way. IIRC Windows Forms is the only .NET UI framework that allows this. WPF and Maui don't.

**Comment by maxkatz6** [comment]
There is no crossplat API for that, but you can get window handle from `window.PlatformImpl.Handle` and call win32 API directly to modify window properties. Similarly, you can do it for other platforms with pinvoke or net6-macos libraries in case of macos.

**Comment by AArnott** [comment]
Duplicate of #3422.

**Comment by harikrishnan-p-v** [comment]
@barnabaspal @AntonNah you can add 
`WindowStartupLocation="CenterOwner"
ExtendClientAreaChromeHints="NoChrome"
ExtendClientAreaToDecorationsHint="True"`
![image](https://github.com/AvaloniaUI/Avalonia/assets/67196336/eb393004-4ed3-4dab-a749-9666dff6c315)
It will hide the default task bar.

**Reply by kirb** [reply]
Note this only hides the button (by hiding the entire title bar), it doesn't prevent minimizing. That can still be done by right-clicking the thumbnail of the window in the taskbar, or pressing Win+M.

**Reply by chuongmep** [reply]
it not a best idea but work well for me incase design two form overlay

**Comment by hansos** [comment]
I’d like to express support for a declarative way to hide the Minimize and Maximize buttons in Avalonia.
A syntax like this would be incredibly convenient and intuitive:


```
<Window Title="Set Password"
        Width="300"
        Height="160"
        ShowMinimizeButton="False"
        ShowMaximizeButton="False"
        CanResize="False"
        WindowStartupLocation="CenterOwner" />
```

This would make it easy to configure modal dialogs or restricted windows without needing platform-specific interop or custom title bars. It would also align well with Avalonia’s goal of being a cross-platform, XAML-friendly UI framework.

I hope this can be considered for future versions — it would be a small but very impactful improvement for developers.
